crates/trigger/src/loader.rs

```diff
@@ -56,6 +56,9 @@ impl Loader for TriggerLoader {
             )
         })?;
         let component = spin_componentize::componentize_if_necessary(&bytes)?;
+        if let Err(e) = std::fs::write(self.working_dir.join("component.wasm"), &component) {
```

Comment:
In a multi-component application, this will (as far as I can tell) result in each component overwriting the previous one.

Reply:
Good catch - unfortunately fixing this won't be easy as we don't have the name of the component accessible at this point. We could use the file name of the module but that's not guaranteed to be unique (e.g., `/some/path/module.wasm` and `/some/other/path/module.wasm). I'll need to noodle a bit on this...